scala-thoughts.md

```diff
@@ -0,0 +1,11 @@
+***What's Easy to do in Scala? What's hard?
+So far I think that I like pretty much everything about Scala. It really incorporates almost all of my favorite features from SML (a languge I initially hated but really really learned to love) while still having OOP and readable error messages (among other improvements). Obviously learning any language is hard and Scala is no exception, but despite my frustrations I really enjoyed my first spin. I think that Scala has a really really steep learning curve. Not an easy language to learn.
```

Comment:
SML is taught in Pomonas equivalent to CS60 (the intro class that is not intro or data structures)